Multisig/UI/Transactions/TransactionDetails/TransactionDetailsView.swift

```diff
@@ -9,26 +9,69 @@
 import SwiftUI
 
 struct TransactionDetailsView: View {
+    
     @FetchRequest(fetchRequest: Safe.fetchRequest().selected())
     var selectedSafe: FetchedResults<Safe>
 
     let transaction: TransactionViewModel
+
+    @State
+    private var showsLink: Bool = false
+    private let padding:CGFloat = 11
+
     var body: some View {
         List {
             TransactionHeaderView(transaction: transaction)
+
+            if data != nil {
+                VStack (alignment: .leading) {
+                    BoldText("Data")
+                    ExpandableButton(title: "\(data!.0) Bytes", value: data!.1)
+                }.padding(.vertical, 11)
+            }
+
             TransactionStatusTypeView(transaction: transaction)
             if !(transaction.confirmations?.isEmpty ?? true) {
-                TransactionConfirmationsView(transaction: transaction, safe: selectedSafe.first!)
+                TransactionConfirmationsView(transaction: transaction, safe: selectedSafe.first!).padding(.vertical, padding)
+            }
+
+            if transaction.createdDate != nil {
```

Comment:
because the createdDate and executedDate are empty strings in the incoming transactions, I've got this screen.

<img src="https://user-images.githubusercontent.com/284056/85005814-cfc86580-b148-11ea-941a-722f9ea72aab.png" width=300/>